CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.8.0 - 12 August 2019
+### Added
+* Changed default behavior in VS Code remote environments to run as a "workspace" extension instead of a "UI" extension. See [#954](https://github.com/Microsoft/vscode-docker/issues/954) for more information
+* Added support to debug ASP.NET Core web apps with SSL enabled
```

Comment:
@bwateratmsft I feel like I'm not fully aware of all that you did for this release - is this line good enough?

Reply:
Yes, I think it's good. No user changes are needed to their launch.json to get things working so this is a freebie for them.